### DISCUSSION TITLE
add in URL in Resource

### DISCUSSION THREAD

**Discussion by msamran27** [filamentphp/filament#6583] [General]
i have DaySlotResource i want to add a button in table view TimeSlots when i click view button then get timeSlots page timeSlots page named is TimeSlotResource kindly help me if it possiable

**Comment by danharrin** [comment]
You probably want to create an action, but I'm not too sure without seeing a screenshot of what you want to do - https://filamentphp.com/docs/2.x/tables/actions#single-actions

**Reply by msamran27** [reply]
![image](https://github.com/filamentphp/filament/assets/100018533/9632c1dd-5c39-4cca-b57b-f6571c95303a)

**Reply by msamran27** [reply]
i Have DaySlot where edit button i remove edit button put TimeSlot when i click TimeSlots then i go to time Slot

**Reply by danharrin** [reply]
Yeah you need an action

**Reply by msamran27** [reply]
okay Thank You
